specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -496,24 +653,32 @@ model ResourceMetric {
   resourceType: string;
 }
 
-@doc("Load test run model")
-@resource("test-runs")
-model TestRuns {
-  // FIXME: is testRunId optional or required in the response? Resource key and path params cannot be optional
-  @key
-  @path
-  @doc("Unique test run name as identifier")
-  @visibility("read")
-  testRunId: string;
-}
+// @doc("Load test run model")
+// @resource("test-runs")
+// model TestRuns {
+//   // FIXME: is testRunId optional or required in the response? Resource key and path params cannot be optional
+//   @key
+//   @path
+//   @doc("Unique test run name as identifier")
+//   @visibility("read")
+//   testRunId: string;
+// }
 
 @doc("Load test run model")
 model TestRun {
-  ...TestRuns,
+  @doc("Unique test run name as identifier")
+  @visibility("read")
+  @maxLength(50)
+  @minLength(2)
+  testRunId?: string;
```

Comment:
Why is it marked optional for TestRun ( isn't is a mandatory property ?

Reply:
This is a little confusing. Ideally this should be a mandatory field and marked as key as it is an identifier to the resource type TestRun. But since we pass it in request body there we marked it as read only. In previous version spec it is optional, so now if we make it required it will change our model. Therefore, we decided to keep it similar to GA version. Although please suggest if we should have some changes here.